tools/__init__.py

```diff
@@ -19,14 +21,19 @@
 
 # Put all tool files in __all__
 # allows "from tools import *" to import all tooles for testtools
-__all__ = [filename[:-3]  # Remove .py
-           for filename in os.listdir(os.path.dirname(__file__))  # tools directory
-           if filename.endswith('.py') and filename != '__init__.py' and filename not in [  # Add any files to exclude here:
-               # e.g. 'sometool.py',
-           ]]
+__all__ = sorted(
+    [
+        filename[:-3]    # Remove .py
+        for filename in os.listdir(os.path.dirname(__file__))    # tools directory
+        if filename.endswith(
+            '.py') and filename != '__init__.py' and filename not in [    # Add any files to exclude here:
+    # e.g. 'sometool.py',
+            ]
+    ]
+)
 installed_tools = {}
 
-LOG = logging.getLogger(__name__)
+log = logging.getLogger(__name__)
```

Comment:
I'm a little curious / confused: at some point in the recent past, there was a sweeping change across the codebase to convert log -> LOG (not sure if this was you, @yesimon, or pylint), and here I see some, but not all, of them reverting back. I don't particularly care either way, so long as it's consistent (but if there's a compelling argument one way or another, let's do that one).